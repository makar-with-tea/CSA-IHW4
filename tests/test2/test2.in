2 1 2 3 test2.out