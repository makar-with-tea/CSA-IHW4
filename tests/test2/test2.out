Покупатель 2 встал в очередь в отделе 1
Покупатель 1 встал в очередь в отделе 3
Отдел 3 обслужил покупателя 1
Отдел 1 обслужил покупателя 2
Покупатель 1 встал в очередь в отделе 1
Покупатель 2 встал в очередь в отделе 2
Отдел 1 обслужил покупателя 1
Отдел 2 обслужил покупателя 2
Покупатель 2 встал в очередь в отделе 1
Отдел 1 обслужил покупателя 2
КОНЕЦ РАБОТЫ МАГАЗИНА
